pyrightconfig.stricter.json

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/pyright/main/packages/vscode-pyright/schemas/pyrightconfig.schema.json",
+    "typeshedPath": ".",
+    "include": [
+        "stdlib",
+        // Modules that pass this stricter mode.
```

Comment:
Sure, I can do it that way.

Reply:
I don't think opt-out is a good idea, at least for third-party packages. I predict that this will cause CI to fail for most new packages. But let's wait and see. We can always flip this back later if it indeed turns out to be a common problem.